examples/llm/vdb_upload/pipeline.py

```diff
@@ -48,6 +49,8 @@ def pipeline(num_threads: int,
              vector_db_resource_name: str,
              triton_server_url: str):
 
+    # WebScraperStage requires C++ mode to be disabled
+    CppConfig.set_should_use_cpp(False)
```

Comment:
May want to hold off on this till the vdb_upload PR goes in, there will substantial changes here. https://github.com/nv-morpheus/Morpheus/pull/1454

Reply:
I'm thinking instead maybe I should just merge my PR into yours since there is a bit of an overlap.